packages/app/src/cli/models/app/app.ts

```diff
@@ -53,7 +53,7 @@ export const LegacyAppSchema = zod
       })
       .optional(),
   })
-  .strict()
+  .strict({message: 'You may be missing client_id, or your TOML is malformed'})
```

Comment:
This solves the local problem, but now if someone is missing a single key from the legacy app schema, they will not know what's wrong. Is that the right tradeoff here?

Reply:
Yeah, I don't think strict here is the right solution.

Also, we should not be using `LegacySchema` almost never, seems like it's only being used "by accident" because if you remove the client-id the CLI thinks the app was never linked and defaults to the `LegacyAppSchema` where everything is optional.